python/cudf/cudf/core/column/lists.py

```diff
@@ -451,3 +457,56 @@ def sort_values(
             sort_lists(self._column, ascending, na_position),
             retain_index=not ignore_index,
         )
+
+    def ravel(self) -> ParentType:
+        """
+        Removes one level of nesting from each row of the list Series.
+
+        Returns
+        -------
+        Series or Index with one level of nesting removed from each row.
+
+        Examples
+        --------
+        >>> s1
+        0      [[1.0, 2.0], [3.0, 4.0, 5.0]]
+        1    [[6.0, nan], [7.0], [8.0, 9.0]]
+        dtype: list
+        >>> s1.list.ravel()
+        0    [1.0, 2.0, 3.0, 4.0, 5.0]
+        1    [6.0, nan, 7.0, 8.0, 9.0]
+        dtype: list
+
+        Null values at the top-level in each row are dropped:
```

Comment:
Is this desirable? If not, what should our behaviour be?

Reply:
Seems ok to me... This may introduce ambiguity of 
```
flatten([[1, 2, 3], None, [4, 5]]
flatten([[1, 2, 3], [], [4, 5]]
```

Even though, when unwrapping nested lists it sounds reasonable to assume both empty list and null item do not contribute to concrete elements.

Reply:
I exposed a `dropna` param which will drop null values by default. If set to `False`, the corresponding row in the result is null.

(these are the options available at the libcudf level)